SpatialGDK/Source/SpatialGDKEditorToolbar/Private/SpatialGDKEditorToolbar.cpp

```diff
@@ -972,7 +972,12 @@ void FSpatialGDKEditorToolbarModule::VerifyAndStartDeployment(FString ForceSnaps
 		}
 
 		FLocalDeploymentManager::LocalDeploymentCallback CallBack = [this](bool bSuccess) {
-			if (!bSuccess)
+			if (bSuccess)
+			{
+				// Once we've launched the local deployment, attempt to start the inspector process
+				StartInspectorProcess(nullptr /*InOnReady*/);
```

Comment:
Omega Nit: I love the comments for ambiguous arguments. I would suggest that maybe I'd prefer simply "OnReady" as I am not sure the "In" adds much. Feel free to ignore though :P

Reply:
nice catch!